src/common/lambda-function.ts

```diff
@@ -18,22 +19,27 @@ export class LambdaFunction {
   public async getFuncVPC (funcName: string, funcVPCDiscovery: FuncVPCDiscovery): Promise<VPC> {
     if (typeof funcVPCDiscovery === "boolean" && !funcVPCDiscovery) {
       // skip vpc setup for `vpcDiscovery=false` option
-      Globals.logInfo(`Skipping VPC config for the function '${funcName}'`);
+      Globals.logInfo(`Skip the VPC config for the function '${funcName}'`);
       return null;
     }
-    const isConfigValid = this.validateVPCDiscovery(funcVPCDiscovery);
-    if (isConfigValid) {
-      // inherit basic config
-      const vpcDiscovery = Object.assign({}, this.basicVPCDiscovery, funcVPCDiscovery);
-      return await this.ec2Wrapper.getVpcConfig(vpcDiscovery);
+    // inherit basic config
```

Comment:
What does `basic config` mean? Is that `custom.vpcDiscovery`? Maybe `global config` would be a better name